src/constants/index.js

```diff
@@ -332,11 +332,13 @@ export const SUPPORTED_LANGUAGES = {
 export const FEATURE_FLAG_LABELS = {
   I18N_ENABLED: 'I18N_ENABLED',
   LEGACY_RT_ENABLED: 'LEGACY_RT_ENABLED',
+  EFB_FORM_SCHEMA_SUPPORT_ENABLED: 'EFB_FORM_SCHEMA_SUPPORT_ENABLED',
```

Comment:
We should have a modification in the file `.env` adding the ff:
```
# Feature flags
REACT_APP_LEGACY_RT_ENABLED=false
REACT_APP_I18N_ENABLED=true
EFB_FORM_SCHEMA_SUPPORT_ENABLED=false
```

Which reminds me... @JoshuaVulcan we should create a ticket to get rid of the `I18N_ENABLED` flag. Should be easy.

Reply:
You mean setting it to false by default? Didn't get it